Source/Core/Common/CommonFuncs.cpp

```diff
@@ -59,11 +62,11 @@ std::string GetLastErrorString()
 // Like GetLastErrorString() but if you have already queried the error code.
 std::string GetWin32ErrorString(DWORD error_code)
 {
-  char error_message[BUFFER_SIZE];
+  TCHAR error_message[BUFFER_SIZE];
```

Comment:
Since this is directly calling `FormatMessageW()` instead of using the `FormatMessage()` macro, shouldn't this use `wchar_t` instead of `TCHAR`?